remove-other-tokens.py

```diff
@@ -0,0 +1,90 @@
+#!/opt/privacyidea/bin/python
+
+from privacyidea.lib.token import get_tokens, get_one_token, remove_token
+from privacyidea.lib.user import User
+import argparse
+from privacyidea.app import create_app
+import logging
+
+__doc__ = """
+This scripts create new tokens for a given user.
```

Comment:
I think the short description is wrong. ;-)
The script actually *removes* all other tokens, but the one that was just created.

Reply:
You are right, I apparently did miss to edit this line.